lib/pipedrive/api_operations/request.rb

```
@@ -17,6 +17,15 @@ def api_version
           supports_v2_api? ? Pipedrive.api_version : :v1
         end
 
+        def api_version_prefix
+          return api_version if api_version == :v1
```

RuboCop (1.73)

[convention] Add empty line after guard clause.

(Layout/EmptyLineAfterGuardClause)